addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/cli/Parser.java

```diff
@@ -341,9 +340,18 @@
      * @return
      */
     private static int parseVolume(String vol) {
-        Matcher matcher = volumePattern.matcher(vol);
-        if (matcher.find()) {
-            return Math.round((Integer.valueOf(matcher.group(3)) + Integer.valueOf(matcher.group(7))) / 2);
+        int volumeTotal = 0, nChannels = 0;
+        for (String channel : vol.split(",")) {
```

Comment:
This won't work mith my setup, because the volume string here is:
` front-left: 30419 /  46% / -20,00 dB,   front-right: 30419 /  46% / -20,00 dB`
so splitting by "," in not a good idea